taskana-adapter-camunda-outbox-rest/src/main/webapp/WEB-INF/weblogic.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<weblogic-web-app xmlns="http://xmlns.oracle.com/weblogic/weblogic-web-app"
+                  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                  xsi:schemaLocation="http://xmlns.oracle.com/weblogic/weblogic-web-app
+                                      http://xmlns.oracle.com/weblogic/weblogic-web-app/1.4/weblogic-web-app.xsd">
+
+  <context-root>/</context-root>
```

Comment:
didn't we agree to habe '/taskana-outbox' as context root?

Reply:
Indeed, context-root is now set to /taskana-outbox for JBOSS and Weblogic